eidpdata/src/main/java/uk/ac/ucl/eidp/data/jaxb/JaxbSqlAnsi.java

```diff
@@ -36,12 +36,16 @@ protected String buildGetStatement() {
         stm.append(String.join(", ", queryFields));
         stm.append(" FROM ").append(tableType.getName());
         
-        if (!methodType.getFor().isEmpty()) stm.append(generateWhereClause());
+        if (!methodType.getFor().isEmpty()) {
```

Comment:
\[Checkstyle\] WARNING: 'if' have incorrect indentation level 8, expected level should be 4\.